### ISSUE TITLE
Create new Fragments from Content Wizard in Content Studio

### ISSUE THREAD

**Issue by Bellfalasch** [enonic/xp#4843]
TL;DR: Add "Fragments" to Create Content Wizard

To create a Fragment today in Content Studio you need to add and configure a part on a page (content) or page template. Then, after the part is done, right-click it and select "Create fragment".

It would be very handy for sites with huge content structures to be able to create the Fragment first. Meaning: navigate to the optimal location in Content Studio, right-click and select "New" and then from the content wizard create a new Fragment and then configure it. Without context. Then they can go to any page and create it.

For larger corporations, like Gjensidige, Posten, this approach works better. They can then have a handful of editors that creates a number of fragments that others can select from. All stored at a logic location within the site's folder structure.

**Comment by mortyeriksen** [comment]
Added to backlog.

To solve this we need a selector, like when selecting a controller for a content, to select available component types.

**Comment by ComLock** [comment]
Combining libContent.getSiteConfig with libContent.getTypes and/or libContent.getType is should already be possible to make a customSelector for this. However I'm guessing your talking about providing a core one.
